community/doc_guidelines/capitalization.md

```diff
@@ -0,0 +1,153 @@
+# Splinter Capitalization Guidelines
+
+Use these guidelines to ensure consistency when writing new Splinter
+documentation and updating existing content.
+
+* Follow the [capitalization rules for American
+  English](https://owl.purdue.edu/owl/general_writing/mechanics/help_with_capitals.html).
+
+* See the [Google style guide word
+  list](https://developers.google.com/style/word-list) for capitalization of
+  standard technical terms.
+
+* For software products, companies, and brands, use the official capitalization
+  (see the official website or check [Wikipedia](http://wikipedia.org)).
+
+* For titles and headings, follow the Chicago Manual of Style's title rules.
+  (Tip: Use [capitalizemytitle.com](https://capitalizemytitle.com/) and select
+  "Chicago".)
+
+    - Capitalize the first and the last word, nouns, pronouns, adjectives,
+      verbs, adverbs, and subordinate conjunctions (such as Because, Once,
+      Therefore, When, and Unless).
+    - Lowercase articles (a, an, the), coordinating conjunctions, prepositions,
+      and the "to" in infinitive verbs.
+
+* Avoid starting a sentence or title with a lowercase term, such as Splinter
+  commands (CLIs), daemons, and services (such as `splinterd`). Instead,
+  rewrite to start with "The" or some other word.
+
+* To form the plural of an all-uppercase term, add a lowercase "s" with no
+  apostrophe, such as CLIs, IDs, and REST APIs.
+
+[A](#a) - [B](#b) - [C](#c) - D - E - F - [G](#g) - H - [I](#i) - [J](#j) -
+[K](#k) - [L](#l) - [M](#m) - [N](#n) - [O](#o) - [P](#p) - Q - [R](#r) -
+[S](#s) - [T](#x) - U - V - [W](#w) - [X](#x) - [Y](#y) - [Z](#z)
+
+## A
+
+* admin service
+* API, APIs
+* application auth handler _(or application authorization handler)_
+
+## B
+
+* Biome
+
+## C
+
+* Canopy
+* CLI, CLIs
+* consensus
+
+## G
+
+* Gameroom _(example application)_
+* gameroom _(circuit in Gameroom)_
+* `gameroomd` _(daemon; always lowercase)_
+* Grid _(at first use: Hyperledger Grid)_
+* `gridd` _(daemon; always lowercase)_
+
+## I
+
+* ID, IDs
+
+## J
+
+* JavaScript
+* JSON
+
+## K
+
+* key registry
```

Comment:
I think key registry can be removed

Reply:
or marked as obsolete

Reply:
Marked this (and Private Counter & XO terms) as obsolete for now.  A later PR is planned to add new terms and remove the obsolete ones.